locale/exchangecalendar/en-US/delegate-calendar-dialog.dtd

```diff
@@ -1,10 +1,10 @@
  
 <!ENTITY  label.delegateCalendar.cancelbutton "Cancel">
 <!ENTITY  label.delegateCalendar.userEmail "Email">
-<!ENTITY  label.delegateCalendar.deliveryMeetingRerquestControl "Deliver Meeting Requests">
+<!ENTITY  label.delegateCalendar.deliveryMeetingRerquestControl "Deliver meeting requests">
 
-<!ENTITY  menuitem.delegateCalendar.deliverMeetingRequestslist.delegateOnly "Delegates Only">
-<!ENTITY  menuitem.delegateCalendar.deliverMeetingRequestslist.delegateAndMe "Delegates and Me">
+<!ENTITY  menuitem.delegateCalendar.deliverMeetingRequestslist.delegateOnly "Delegates only">
```

Comment:
Fixed in Commit 23ddbe79842f69dcadc781d31cb2cfa4b3cb0fe1.  I guess these are the only menu items that need to be changed…?  I think the only other `menuitem` entities I changed were `menuitem.label` ones that seem to use sentence case rather than title case.